vernac/g_vernac.mlg

```diff
@@ -321,10 +322,13 @@ GRAMMAR EXTEND Gram
       | -> { None } ] ]
   ;
   decl_notation:
-    [ [ ntn = ne_lstring; ":="; c = constr; b = only_parsing;
+    [ [ ntn = ne_lstring; ":="; c = constr;
+        modl = [ "("; l = LIST1 syntax_modifier SEP ","; ")" -> { l } | -> { [] } ];
```

Comment:
Shouldn't we factor this parenthesized modifier list? For instance the equations overlay ends up with a space-separated list, if it was factored it would just get the same thing.

Reply:
Done!